### PR TITLE
Update mysql.md

### PR DESCRIPTION
**Pull request by okke-formsma**
Added an example with a database on a different host.

### CODE REVIEW

**Comment by ostera** [comment]
If mysql assumes the database is local when no `-h` is specified, we should change the text to read `connect to a local database` an so on.

**Comment by igorshubovych** [comment]
I would split host and password examples

File: pages/common/mysql.md [hunk 1]
```diff
@@ -8,7 +8,11 @@
 
 - Connect to a database using credentials
 
-`mysql -u {{user}} -p{{password}} {{database_name}}`
+`mysql -h {{database_host}} -u {{user}} -p{{password}} {{database_name}}`
+
+- Connect to a database on a different host. MySQL will prompt for a password if none is supplied.
```
Comment (denis-sokolov) [inline pages/common/mysql.md]: The wording "on a different host" is clunky in comparison to the previous command. It seems it tries to contradict, but that's not the case. Perhaps "Connect by entering the password interactively"?

**Comment by igorshubovych** [comment]
@okke-formsma are you going to continue working on this?

**Comment by okke-formsma** [comment]
@igorshubovych I'll take a look tonight (CET)

**Comment by igorshubovych** [comment]
Thank you for your contribution!

**Pull request merged**